Tests/SourceKitLSPTests/DocumentTestDiscoveryTests.swift

```diff
@@ -210,14 +210,14 @@ final class DocumentTestDiscoveryTests: XCTestCase {
       tests,
       [
         TestItem(
-          id: "MyTests",
+          id: "MyTests/test.swift:2:0",
           label: "MyTests",
           disabled: false,
           style: TestStyle.swiftTesting,
           location: Location(uri: uri, range: positions["1️⃣"]..<positions["4️⃣"]),
           children: [
             TestItem(
-              id: "MyTests/oneIsTwo()",
+              id: "MyTests/oneIsTwo()/test.swift:3:2",
```

Comment:
The location of a test is the start of the `Test` identifier in `@Test`. You can see this because `swift test --filter 'MyTests/oneIsTwo\(\)/test.swift:3:2'` doesn’t actually run any tests in this project (or rather, if you create an equivalent project that actually compiles because it has a proper `swift-testing` dependency declaration).